webapp/src/main/webapp/templates/freemarker/edit/forms/js/manageHideShowStatus.js

```diff
@@ -64,7 +64,7 @@ var manageHideShowStatus = {
                 additions: add,
                 retractions: retract
             },
-            dataType: 'json',
+            dataType: 'text',
```

Comment:
Please check this

Reply:
Checked. The endpoint doesn't return json, so it should be text.